mobile/src/libs/components/modal/modal.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Modal as RNModal } from 'react-native';
+
+import { Icon, Pressable, View } from '#libs/components/components';
+import { AppColor } from '#libs/enums/enums';
+
+import { styles } from './styles';
+
+type Properties = {
+  isVisible: boolean;
+  onClose: () => void;
+  onDelete?: () => void;
+  children?: React.ReactNode;
+};
+
+const Modal: React.FC<Properties> = ({
+  children,
+  isVisible,
+  onClose,
+  onDelete,
+}) => {
+  return (
+    <RNModal visible={isVisible} transparent={true} animationType="none">
+      <Pressable onPress={onClose} style={styles.background}>
+        <View style={styles.modalContainer}>
+          <Pressable>
+            <View style={styles.modalView}>
+              {onDelete ? (
+                <></>
```

Comment:
Why don't we display anything if there's `onDelete`?

Reply:
Because there is @iamAlinaaa 's code there. I decided not to add mine because this piece of code will not be used anywhere in my part